src/components/Widgets/WalletButton.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+export default class WalletButton extends Component {
+  static displayName = 'WalletButton'
+
+  static propTypes = {
+    /** Display text to be displayed as button symbol */
+    buttonSymbol: PropTypes.string,
+    /** Display text describing button action */
+    innerText: PropTypes.string,
+    /** Callback to be executed onClick */
+    onClick: PropTypes.func
+  }
+
+  static defaultProps = {
+    buttonSymbol: '+',
+    innerText: ''
+  }
+
+  handleClick = e => {
+    const { onClick } = this.props
+    if (onClick) {
+      onClick(e)
+    }
+  }
+
+  render() {
+    const { buttonSymbol, innerText } = this.props
+    return (
+      <StyledButton onClick={this.handleClick}>
+        <StyledDiv> {buttonSymbol} </StyledDiv>
+        <StyledH3> {innerText} </StyledH3>
```

Comment:
Instead of `innerText`, lets allow a user to pass anything through, e.g. `children`. The thinking is that this API should more closely resemble the other Button API. See: https://github.com/ethereum/ethereum-react-components/blob/master/src/components/Widgets/Button.jsx#L41

```
const { children } = this.props

...
<StyledH3>{children}</StyledH3>
...
```
usage:
```
<WalletButton {...this.props} onClick={() => {}}>Add Wallet Contract</WalletButton>
```

Reply:
done.